src/cpu/src/opcodes.cpp

```diff
@@ -376,14 +376,15 @@ void CPU::op_rlc(IReg *reg) {
 
 void CPU::op_rlc(Address addr) {
 	auto value = memory->read(addr);
-	auto msb = static_cast<bool>(value >> 7);
+	bool msb = value & (1 << 7);
```

Comment:
Can you explain what have you exactly done here?

Reply:
This opcode was setting the zero flag incorrectly, so I thought I'd make a quick fix for that. This line change is just to make it a tad more readable.

Reply:
Got it.